release/sil/sil.kjg-laoo.ptwl1/.gitignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
You should be able to remove this file, since this is covered by the .gitignore file at the root of this repo
`release/*/*/build/`